builtin/settingtypes.txt

```diff
@@ -1046,6 +1046,10 @@ client_unload_unused_data_timeout (Mapblock unload timeout) int 600
 #    Set to -1 for unlimited amount.
 client_mapblock_limit (Mapblock limit) int 7500
 
+#    Whether to show technical names for mods, texture packs and settings.
+#    Controlled by the checkbox in the "All settings" menu.
+main_menu_technical_settings (Show technical names) bool false
```

Comment:
You are taking a setting here that is already used by existing code, please document its original purpose too.

Reply:
It does say "and settings", but I'll try to make it clearer.